### DISCUSSION TITLE
Is there a way to toggle the visibility of the filter row?

### DISCUSSION THREAD

**Discussion by younus93** [Power-Components/livewire-powergrid#850] [Q&A]
Hi, 

I am working with a product which will be used predominently on a small screen. Is there any way to have a button which can toggle the visibility of the filters (OR) show the filters in a modal/dropdown?

**Comment by luanfreitasdev** [comment]
You can use the 'outside' filter

https://livewire-powergrid.com/get-started/configure.html?id=configure#_7-filters

**Reply by younus93** [reply]
Thanks for your reply. 
Is there a way to customize this view? Can we persist the toggled state like how we persist the search / column visibility?
